packages/web-components/src/components/card-link/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import ExpressiveModal from './ExpressiveModal';
+
+# Card Link
+
+<Preview withToolbar>
+  <Story id="components-card-link--default" height="120px" />
+</Preview>
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/card-link)
+> example implementation.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/card-link/card-link';
+import ArrowRight20 from 'carbon-custom-elements/es/icons/arrow--right/20';
+```
+
+##### HTML
+
+```html
+<dds-card-link href="https://example.com">
+  <p>Copy text</p>
+  ${ArrowRight20({ slot: 'footer' }}
```

Comment:
`${ArrowRight20({ slot: 'footer' })}` is `lit-html` specific, probably we should put raw `<svg slot="footer">`?

Reply:
Sounds good